action.yml

```diff
@@ -62,5 +67,8 @@ runs:
         if [[ -n ${{ env.BASE_BRANCH }} ]]; then
           BASE_BRANCH_OPTION="--base ${{ env.BASE_BRANCH }}"
         fi
-        ${{ github.action_path }}/create_tag.py --token "${{ inputs.TOKEN }}" $VERSION_TYPE_OPTION $BASE_BRANCH_OPTION
+        if [[ -n ${{ env.VERSION }} ]]; then
+          VERSION_OPTION="--version ${{ env.VERSION }}"
```

Comment:
Is there a potential code execution exploit here?  If the version is `v1.0;./leak-all-secrets` or `"1.1;base64 <<< $GITHUB_TOKEN"`?

I'm assuming the action input field is free-form.

Reply:
I used the intermediate env variable approach to be on the safe side. The `version` input is indeed a free-form string, so it could be anything.